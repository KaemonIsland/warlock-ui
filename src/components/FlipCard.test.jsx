import renderer from 'react-test-renderer'
import React from 'react'
import { FlipCard, CardSide } from '.'
import { ThemeProvider } from '..'

// prettier-ignore
test('The FlipCard works!', () => {
  const tree = renderer
    .create(
      <ThemeProvider>
        <FlipCard>
          <CardSide side="front">Front</CardSide>
          <CardSide side="back">Back</CardSide>
        </FlipCard>
      </ThemeProvider>,
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
