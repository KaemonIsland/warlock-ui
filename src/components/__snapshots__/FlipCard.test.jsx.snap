// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The FlipCard works! 1`] = `
<div
  className="FlipCard__StyledFlipCard-sc-1p0b7wp-0 liCTAA"
  onClick={[Function]}
>
  <div
    className="FlipCard__StyledCardSide-sc-1p0b7wp-1 fMEwyZ"
  >
    Front
  </div>
  <div
    className="FlipCard__StyledCardSide-sc-1p0b7wp-1 fIdZxG"
  >
    Back
  </div>
</div>
`;
